### PR TITLE
Initial YAML reference chewed open

### PR DESCRIPTION
**Pull request by jnummelin**
Signed-off-by: Jussi Nummelin <jnummelin@mirantis.com>

### CODE REVIEW

**Comment by jnummelin** [comment]
Need to still add `metadata` stuff

File: docs/configuration-file.md [hunk 1]
```diff
@@ -45,3 +45,54 @@ spec:
...
+
+## `metadata`
+
+- `name` - Name of the cluster to be created. Affects only `launchpad` internal storage paths currently e.g. for client bundles.
+
+## `spec`
+
+The specification for the cluster.
+
+## `hosts`
```
Comment (jakolehm) [inline docs/configuration-file.md]: Maybe indent stuff under `spec`?
```suggestion
### `hosts`
```

Comment (jnummelin) [reply docs/configuration-file.md]: done

File: docs/configuration-file.md [hunk 2]
```diff
@@ -45,3 +45,54 @@ spec:
...
+- `privateInterface` - Discover private network address from the configured network interface (optional)
+- `role` - One of `manager` or `worker`, specifies the role of the machine in the cluster
+
+### `ucp`
+
+Specify options for UCP cluster itself.
+
+- `version` - Which version of UCP we should install or upgrade to (default `3.3.0`)
+- `imageRepo` - Which image repository we should use for UCP installation (default `docker.io/docker`)
+- `installFlags` - Custom installation flags for UCP installation. You can get a list of supported installation options for a specific UCP version by running the installer container with `docker run -t -i --rm docker/ucp:3.3.0 install --help`.
```
Comment (nevalla) [inline docs/configuration-file.md]: How about pointing to docs url also for installFlags?

Comment (nevalla) [inline docs/configuration-file.md]: should we put `(optional)` to all options that does not have default value are not required?

File: docs/getting-started.md [hunk 3]
```diff
@@ -82,4 +82,15 @@ The `launchpad` tool connects to the infrastructure you've specified in the `clu
 
 ## Interact with your cluster
 
-TBD
+At the end of the installation procedure, launchpad will show you the details you can use to connect to your cluster. You will see something like this:
+```
+INFO[0021] ==> Running phase: UCP cluster info 
+INFO[0021] Cluster is now configured. You can access your cluster admin UI at: https://jari-ucp-cluster-master-lb-895b79a08e60c67b.elb.eu-north-1.amazonaws.com
```
Comment (nevalla) [inline docs/getting-started.md]: infamous `jari-ucp-cluster`, what happened to Jorma? 😆

Comment (jnummelin) [reply docs/getting-started.md]: good point, I'll change this

File: docs/configuration-file.md (hunk 2 above)
Comment (jnummelin) [reply docs/configuration-file.md]: I can't find those documented anywhere on web, hence this way of getting all the flags.

**Comment by jnummelin** [comment]
@nevalla fixed all things fixable, PTAL

File: docs/configuration-file.md (hunk 2 above)
Comment (nevalla) [reply docs/configuration-file.md]: I found for 3.2 this https://docs.docker.com/reference/ucp/3.2/cli/install/#options. Don't know do we have for 3.3.0 anything similar. We can add that later if we find it.

**Review by nevalla — APPROVED** [review]
LGTM

**Review by jakolehm — APPROVED** [review]

**Pull request merged**